conf.py

```diff
@@ -110,6 +110,7 @@ def _asset_hash(path: os.PathLike[str]) -> str:
 rediraffe_redirects = {
     # Development Tools
     "clang.rst": "development-tools/clang.rst",
+    "clinic.rst": "development-tools/clinic.rst",
```

Comment:
These redirects were for when pages were at the top level -- it isn't really needeed for an entirely new page.

Reply:
Ok, thanks!